packages/desktop-client/src/components/schedules/SchedulesTable.tsx

```diff
@@ -49,22 +85,33 @@ function OverflowMenu({ schedule, status, onAction }) {
           onClose={() => setOpen(false)}
         >
           <Menu
-            onMenuSelect={name => {
+            onMenuSelect={(name: ScheduleItemAction) => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we should be able to automatically infer this type from `Menu` arguments

Reply:
The inference is a `string`, I added a type to enforce it to a more strict one. If you think, it is too much of a type definitions explicitly defined, I can remove it so to infer to `string`